engine/api/src/main/scala/pl/touk/nussknacker/engine/api/definition/WithExplicitMethodToInvoke.scala

```diff
@@ -21,7 +21,7 @@ trait WithExplicitMethodToInvoke {
 
   def returnType: TypingResult
 
-  def realReturnType: TypingResult
+  def realReturnType: Class[_]
```

Comment:
Please also rename all `realReturnType` fields to `runtimeClass`. Just like in `Parameter` class.

Reply:
:+1: , done